Yosemite/Yosemite/Model/Payments/PaymentIntent+ReceiptParameters.swift

```diff
@@ -0,0 +1,17 @@
+import Hardware
+
+extension PaymentIntent {
+    /// Maps a PaymentIntent into an struct that contains only the data we need to
+    /// render a receips.
+    /// - Returns: an optional struct containing all the data that needs to go into a receipt
+    func receiptParameters() -> ReceiptParameters? {
```

Comment:
We probably will need order ID too so that a receipt can access the items in the order?

Reply:
Nevermind. I see the action has the orderID already